pip/compat.py

```diff
@@ -41,7 +46,14 @@ def console_to_str(s):
         try:
             return s.decode(sys.__stdout__.encoding)
         except UnicodeDecodeError:
-            return s.decode('utf_8')
+            if WINDOWS:
+                try:
+                    from ctypes import cdll
+                    return s.decode("cp" + str(cdll.kernel32.GetACP()))
+                except ImportError:
```

Comment:
I'm not familiar with IronPython, so I don't know if the ctypes module could be imported properly.
So I just add the try-catch to prevent more error.
If it's unnecessary, I'm glad to remove it.